python/docs/source/index.md

```diff
@@ -3,94 +3,116 @@ hide-toc: true
-  content: "Quickly process events so you can respond in real-time."
-  link: ".#stream"
-- header: "{fab}`python;pst-color-primary` Python-native"
-  content: "Use Python so you can load data, process it, and train and serve models from one place."
-  link: ".#python"
-- header: "{fas}`gauge-high;pst-color-primary` Get started immediately"
-  content: "No infrastructure to provision let's you jump right in."
-  link: ".#get-started"
-
-- header: "{fas}`fast-forward;pst-color-primary` Real-time, Batch and Streaming"
-  content: "Execute large-historic queries or materialize in real-time. Or both."
-  link: ".#real-time-and-historic"
-- header: "{fas}`rocket;pst-color-primary` Local, Remote and Distributed"
-  content: "Develop and test locally. Deploy to Docker, K8s or a service for production."
-  link: ".#local-and-distributed"
-- header: "{fas}`backward;pst-color-primary` Time-travel"
-  content: "Generate training examples from the past to predict the future."
-  link: ".#time-travel"
+- header: "{fas}`timeline;pst-color-primary` Real-time Aggregation"
+  content: "Precompute model inputs from streaming data with robust data connectors, transformations & aggregations."
+- header: "{fas}`binoculars;pst-color-primary` Event Detection"
+  content: "Trigger pro-active AI behaviors by identifying significant changes in your data, when they happen."
+- header: "{fas}`backward;pst-color-primary` History Replay"
+  content: "Backtest and fine-tune from historical data using per-example time travel and point-in-time joins."
 ```
 
-* * *
-
-(stream)=
-# Real-time event-processing
 
-Kaskada is built on Apache Arrow, providing an efficient, columnar representation of data.
-The same approach is at the core of many analytic databases as well as Pandas and Polars.
+### Real-time AI in minutes
 
-Kaskada goes beyond the columnar representation, by introduce a Timestream -- a columnar representation of events, ordered by time and grouped by key.
-This representation is a perfect fit for all kinds of events, modern event streams as well as events stored in a database.
-Specializing for Timestreams allows Kaskada to optimize temporal queries and execute them much faster.
+Connect and compute over databases, streaming data, _and_ data loaded dynamically using Python..
+Kaskada is seamlessly integrated with Python's ecosystem of AI/ML tooling so you can load data, process it, train and serve models all in the same place.
 
-(python)=
-# Python-native
+There's no infrastructure to provision (and no JVM hiding under the covers), so you can jump right in - check out the [Quick Start](quickstart).
 
-Connect to existing data in streams or databases, or load data using Python.
-Wherever your events are stored, Kaskada can help you process them.
 
-Build temporal queries and process the results using Python.
-Connect straight to your visualizations, dashboards or machine learning systems.
+(scalable-and-reliable)=
```

Comment:
Maybe drop this anchor as well? Don't see it being used.

Reply:
doh